certif/app/controllers/authenticated/sessions/add-student.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable ember/no-computed-properties-in-native-classes*/
+
+import Controller from '@ember/controller';
+import { alias } from '@ember/object/computed';
+
+export default class SessionsAddStudentController extends Controller {
+
+  @alias('model.session') session;
+
+  get pageTitle() {
+    let title = 'Inscription des candidats';
+    title += ' | Session ' + this.session.id;
+    title += ' | Pix Certif';
+    return title;
```

Comment:
Dans ce commit on pourrait utiliser des templates pour simplifier et rendre le code plus lisible (à mon avis) : 

```suggestion
return `Inscription des candidats | Session ${this.session.id} | Pix Certif`;
```